Rawl_B502/mínugud_senchusa_laigin_and_so_sís.ttl

```diff
@@ -0,0 +1,1264 @@
+    rdfs:comment "Is Cobthach Cóel ro h-ort Lóegaire Lorcc h-i tosuch conid ó h-shain le is chocad eter a clanna .i. Lagin & Leth Cuind ut Orthanach loquitur.".
+
+<#LóegaireLorcc>
+    a foaf:Person;
+    irishRel:nomName "Lóegaire Lorcc";
+    owl:sameAs <#LóegaireLuirc>.
+
+<#Orthanach>
+    a foaf:Person;
+    irishRel:nomName "Orthanach";
+    rdfs:seeAlso <http://vanhamel.nl/codecs/Orthanach_%C3%BAa_C%C3%B3ill%C3%A1ma>;
+    rdfs:comment "[author of 'Cóic cét bliadnae buadach rím']".
+
+<#Ferchertne>
+    a foaf:Person;
+    irishRel:nomName "Ferchertne".
+
+# owl:sameAs <http://example.com/Rawl_B502/senchas_síl_h_ir_fo_h_érind.ttl#FerchertneFile>?
+# They are both called Ferchertne and they are both poets - that is all! EPT
+
+<http://www.vanhamel.nl/codecs/Dind_R%C3%ADg,_r%C3%BAad_T%C3%BAaim_Tenbath>
+    prov:wasAttributedTo <#Ferchertne>.
+
+<#Labraid-4d52ed70>
+    a foaf:Person;
+    irishRel:nomName "Labraid";
+    rel:descendantOf <#LuirccLóegaire>;
+    rel:antagonistOf <#Lugaid>;
+    rel:antagonistOf <#Sétna>;
+    rel:antagonistOf <#CoelCobthach>;
+    owl:sameAs <#LabraidLoingsechMóen>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Dind_R%C3%ADg,_r%C3%BAad_T%C3%BAaim_Tenbath>.
+
+<#LuirccLóegaire>
+    a foaf:Person;
+    irishRel:genName "Luircc Lóegaire";
+    irishRel:nomName "Lorcc Lóegaire";
+    owl:sameAs <#LóegaireLorcc>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Dind_R%C3%ADg,_r%C3%BAad_T%C3%BAaim_Tenbath>.
+
+<#Lugaid>
+    a foaf:Person;
+    irishRel:nomName "Lugaid";
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Dind_R%C3%ADg,_r%C3%BAad_T%C3%BAaim_Tenbath>.
+
+<#Sétna>
+    a foaf:Person;
+    irishRel:nomName "Sétna";
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Dind_R%C3%ADg,_r%C3%BAad_T%C3%BAaim_Tenbath>.
+
+<#CoelCobthach>
+    a foaf:Person;
+    irishRel:nomName "Coel Cobthach";
+    owl:sameAs <#CobthachCóel>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Dind_R%C3%ADg,_r%C3%BAad_T%C3%BAaim_Tenbath>.
+
+<#Augaine>
+    a foaf:Person;
+    irishRel:nomName "Augaine";
+    owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.ttl#AugaineMár>;
+    owl:sameAs <http://example.com/LL/comuammand_na_n-genelach.ttl#UgaineMár>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Dind_R%C3%ADg,_r%C3%BAad_T%C3%BAaim_Tenbath>.
+
+<http://www.vanhamel.nl/codecs/Ní_celt_ceis>
+    prov:wasAttributedTo <#Find>;
+    rdfs:comment "amal ad-fét Find m. Rossa isin fursunnud do-rigni".
+
+<#Find>
+    a foaf:Person;
+    irishRel:nomName "Find";
+    rel:childOf <#Rossa>;
+    owl:sameAs <http://example.com/Rawl_B502/bunad_laigen.ttl#Find>.
+
+<#Rossa>
+    a foaf:Person;
+    irishRel:nomName "Ros";
+    irishRel:genName "Rossa";
+    owl:sameAs <http://example.com/Rawl_B502/bunad_laigen.ttl#RossaRuaid>.
+
+<#Labraid-b9133180>
+    a irishTitle:Rí ;
+    irishRel:nomName "Labraid";
+    owl:sameAs <#Labraid-4d52ed70>;
+    rdfs:seeAlso <http://www.vanhamel.nl/codecs/Ní_celt_ceis>;
```

Comment:
In the *poem, apparently. The poem isn't given in full (first couplet, then "et rel") in Rawl.B.502 but it is cited in illustration of Labraid Loingsech's career, so presumably they appear (and they do certainly appear in full versions of this poem elsewhere).